content/r-package-dev/Documentation.Rmd

```diff
@@ -262,11 +304,17 @@ etc ...
 
 ```
 
-Save your new R Markdown file. The next time you build your package, your vignette should now be available!
+Save your new R Markdown file. To get the vignette to show up in your R package, you then need to:
+
+```{r eval=FALSE}
+devtools::build_vignettes()
+```
+
+This will create the html output, an R script that is just the executable R code in the vignette, and a copy of the markdown file, moving all 3 of those files into the inst/doc folder.
```

Comment:
something like that I guess. I find it odd that so many people in our own group can't seem to remember to build the vignettes properly and check them in...it's clearly something we need to document better. 90% of the time, people just edit the file in the vignette folder, and the "real" vignette doesn't get updated.